### PR TITLE
Minz session cookie path bug (nginx?)

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/924
Probably too late for 1.1.2-beta? https://github.com/FreshRSS/FreshRSS/issues/919

### CODE REVIEW

File: lib/Minz/Session.php [hunk 1]
```diff
@@ -66,9 +66,11 @@ public static function unset_session($force = false) {
...
 	public static function keepCookie($l) {
 		// Get the script_name (e.g. /p/i/index.php) and keep only the path.
-		$cookie_dir = empty($_SERVER['SCRIPT_NAME']) ? '' : $_SERVER['SCRIPT_NAME'];
-		$cookie_dir = dirname($cookie_dir);
-		session_set_cookie_params($l, $cookie_dir, '', false, true);
+		$cookie_dir = empty($_SERVER['REQUEST_URI']) ? '/' : $_SERVER['REQUEST_URI'];
+		if (substr($cookie_dir, -1) !== '/') {
+			$cookie_dir = dirname($cookie_dir) . '/';
+		}
+		session_set_cookie_params($l, $cookie_dir, '', false, false);
```
Comment (marienfressinaud) [inline lib/Minz/Session.php]: Why did you change `true` into `false` for `httponly`?

Comment (Alkarex) [inline lib/Minz/Session.php]: It was a mistake. Corrected https://github.com/Alkarex/FreshRSS/commit/189e790f32d4e389cf1dc6da669a579717fff436
